### ISSUE TITLE
Requirements.txt specifies dataclasses==0.8 

### ISSUE THREAD

**Issue by PS-AI** [harlanhong/CVPR2022-DaGAN#8]
There is an error when installing with requirements.txt. Does the version of dataclasses need to be changed?

**Comment by harlanhong** [comment]
Sorry, you can try to remove the dependencies in requirements.txt that cause errors. The dependency "dataclasses" does not seem to be used in this repo.

**Comment by harlanhong** [comment]
I will upload a clean version of the requirements.txt soon

**Comment by tushar-31093** [comment]
What is the purpose of KP and what all parameters can it take?

**Comment by harlanhong** [comment]
Hi @tushar-31093 ,

Do you mean the keypoint detector? If yes, the keypoint detector aims to capture the motion (expression, pose) of the given face and represent the motion as a set of keypoints. In our experiments, we have tried a set of variants of the keypoint detector, thus we maintain the parameter "kp_detector" in run.py. However, we removed those useless variants and set the useful one as default.

**Comment by PS-AI** [comment]
> 

I was able to run the application after doing these changes to requirements.txt:
absl-py==0.15.0
argon2-cffi==21.1.0
async-generator==1.10
attrs==21.2.0
backcall==0.2.0
bleach==4.1.0
blessings==1.7
cached-property==1.5.2
cachetools==4.2.4
certifi==2021.5.30
cffi==1.14.0
chardet==4.0.0
charset-normalizer==2.0.4
clean-fid==0.1.15
click==8.0.1
cPython==0.0.6
cycler==0.10.0
dataclasses==0.6
decorator==4.4.2
deepdish==0.3.7
defusedxml==0.7.1
dill==0.3.4
dominate==2.6.0
entrypoints==0.3
face-alignment==1.3.4
facenet-pytorch==2.5.2
fancycompleter==0.9.1
google-auth==2.3.3
google-auth-oauthlib==0.4.6
gpustat==0.6.0
GPUtil==1.4.0
grpcio==1.41.1
h5py==3.1.0
idna==2.10
imageio==2.9.0
imageio-ffmpeg==0.4.5
importlib-metadata==4.8.1
ipykernel==5.5.5
ipython==7.16.1
ipython-genutils==0.2.0
ipywidgets==7.6.4
jedi==0.18.0
Jinja2==3.0.1
joblib==1.0.1
jsonpatch==1.32
jsonpointer==2.1
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==7.0.2
jupyter-console==6.4.0
jupyter-core==4.7.1
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.0.1
kiwisolver==1.3.1
llvmlite==0.36.0
Markdown==3.3.4
MarkupSafe==2.0.1
matplotlib==3.3.4
mistune==0.8.4
mkl-fft==1.3.0
mkl-random==1.2.0
mkl-service==2.3.0
nbclient==0.5.4
nbconvert==6.0.7
nbformat==5.1.3
nest-asyncio==1.5.1
networkx==2.5.1
nibabel==3.2.1
nilearn==0.8.1
nltools==0.4.5
notebook==6.4.3
numba==0.53.1
numexpr==2.7.3
numpy==1.19.5
nvidia-ml-py3==7.352.0
oauthlib==3.1.1
olefile==0.46
opencv-contrib-python==4.5.4.58
opencv-python==4.5.3.56
#openface==0.2.1
packaging==21.0
pandas==1.1.5
pandocfilters==1.4.3
parso==0.8.2
pdbpp==0.10.3
pexpect==4.8.0
pickleshare==0.7.5
Pillow==8.3.1
piq==0.5.5
prometheus-client==0.11.0
prompt-toolkit==3.0.19
protobuf==3.17.3
psutil==5.8.0
ptyprocess==0.7.0
py-feat==0.3.7
pyasn1==0.4.8
pyasn1-modules==0.2.8
Pygments==2.9.0
pymongo==3.12.1
pynv==0.2
pyparsing==2.4.7
pypdb==2.0
pyrepl==0.9.0
pyrsistent==0.18.0
python-dateutil==2.8.2
pytorch-msssim==0.2.1
pytz==2021.1
PyWavelets==1.1.1
PyYAML==5.4.1
pyzmq==22.2.1
qtconsole==5.1.1
QtPy==1.11.0
requests==2.25.1
requests-oauthlib==1.3.0
rsa==4.7.2
scikit-image==0.18.1
scikit-learn==0.24.2
scipy==1.5.4
seaborn==0.11.2
Send2Trash==1.8.0
sklearn==0.0
tables==3.6.1
tensorboard==1.15.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
terminado==0.11.1
testpath==0.5.0
threadpoolctl==2.2.0
tifffile==2020.9.3
timm==0.4.12
torch
torchaudio==0.9.0
torchfile==0.1.0
torchvision
tornado==6.1
tqdm==4.62.0
traitlets==4.3.3
typing-extensions==3.10.0.0
urllib3==1.26.6
visdom==0.1.8.9
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.2.1
Werkzeug==2.0.2
widgetsnbextension==3.5.1
wmctrl==0.4
zipp==3.5.0
